coordinator/src/test/scala/com/socrata/datacoordinator/service/collocation/CoordinatedCollocatorTest.scala

```diff
@@ -299,6 +299,32 @@ class CoordinatedCollocatorTest extends FunSuite with Matchers with MockFactory
     }
   }
 
+  test("explainCollocation for one default store group and a collocated dataset and a non-collocated dataset should move the non-collocated dataset twice") {
```

Comment:
It would be nice to have a simple illustration of where the dataset will be moving. Just something like:
`bravo2: store5A -> store1A`
or whatever.